doc/api/fs.md

```diff
@@ -181,6 +181,13 @@ added: v0.1.93
 Emitted when the `ReadStream`'s underlying file descriptor has been closed
 using the `fs.close()` method.
 
+### readStream.bytesRead
+<!-- YAML
+added: v6.4.0
```

Comment:
I’d say with 99 % confidence that the version here is accurate, but ideally you can use `added: REPLACEME` here so that the correct version will be inserted when the release is made.